Apps/Encounter/src/appsettings.hgmock.json

```diff
@@ -7,6 +7,15 @@
     "OpenIdConnect": {
         "Authority": "https://dev.oidc.gov.bc.ca/auth/realms/ff09qn3f"
     },
+    "ODR": {
+        "ServiceName": "ODRPROXY_SERVICE",
+        "ServiceHostSuffix": "_HOST",
+        "ServicePortSuffix": "_PORT",
+        "DynamicServiceLookup": "false",
+        "BaseEndpoint": "https://mock.healthgateway.gov.bc.ca/mockservice/v1/api/odr/",
```

Comment:
should be /api/mockservice which is what our deployment scripts are standardized on.